docs/getting-started.rst

```diff
@@ -152,6 +168,33 @@ With ``--stop-on-failure``
 
   Result: Failed
 
+Charm mapping
+"""""""""""""
+
+This option enables user to explicitly tell ``juju-verify`` which charm a
+specific application deploys.
+
+By default, ``juju-verify`` parses URL from which the charm was deployed to
+identify the charm. However this may fail if charm was deployed from local
+source or from non-official charmstore repository. In such cases, this option
+can be used to specify which charm is an application deploying.
+
+For example if you'd deploy ``ceph-osd`` charm to ``ceph-osd-ssd`` application
+**from local source**, you could do following command to verify it:
```

Comment:
Use of impersonal mode: `For example, if the ``ceph-osd`` charm uses a local path to deploy the ``ceph-osd-ssd`` application, the following command could be used to verify units of the mentioned application:`